test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -846,5 +848,52 @@ public void TensorFlowTransformCifarInvalidShape()
             }
             Assert.True(thrown);
         }
+
+        /// <summary>
+        /// Class to hold features and predictions.
+        /// </summary>
+        public class TensorFlowSentiment
+        {
+            public string Sentiment_Text;
+            [VectorType(600)]
+            public int[] Features;
+            [VectorType(2)]
+            public float[] Prediction;
+        }
+
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]
+        public void TensorFlowSentimentClassificationTest()
```

Comment:
The test is going to fail as Microsoft.ML.TensorFlow.TestModels nuget is not updated yet.